<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="288" started-at="2018-01-28T14:18:26Z" finished-at="2018-01-28T14:18:26Z">
    <groups>
    </groups>
    <test name="ExceptionTests" duration-ms="118" started-at="2018-01-28T14:18:26Z" finished-at="2018-01-28T14:18:26Z">
      <class name="parser.JsonParserTest">
        <test-method status="PASS" signature="initialize()[pri:0, instance:parser.JsonParserTest@3aa9e816]" name="initialize" is-config="true" duration-ms="86" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="readFromFileTest(java.lang.String)[pri:0, instance:parser.JsonParserTest@3aa9e816]" name="readFromFileTest" duration-ms="6" started-at="2018-01-28T08:18:26Z" data-provider="files" finished-at="2018-01-28T08:18:26Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
The data provider is trying to pass 2 parameters but the method parser.JsonParserTest#readFromFileTest takes 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method parser.JsonParserTest#readFromFileTest takes 1
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readFromFileTest -->
        <test-method status="PASS" signature="cleanCart()[pri:0, instance:parser.JsonParserTest@3aa9e816]" name="cleanCart" is-config="true" duration-ms="0" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanCart -->
      </class> <!-- parser.JsonParserTest -->
    </test> <!-- ExceptionTests -->
    <test name="Regression1" duration-ms="6" started-at="2018-01-28T14:18:26Z" finished-at="2018-01-28T14:18:26Z">
      <class name="shop.VirtualItemTest">
        <test-method status="PASS" signature="verifyThatToStringReturnsCorrectResultTest()[pri:0, instance:shop.VirtualItemTest@66d33a]" name="verifyThatToStringReturnsCorrectResultTest" duration-ms="0" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyThatToStringReturnsCorrectResultTest -->
      </class> <!-- shop.VirtualItemTest -->
      <class name="shop.CartTest">
        <test-method status="PASS" signature="initialize()[pri:0, instance:shop.CartTest@45c8e616]" name="initialize" is-config="true" duration-ms="0" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="verifyThatGetPriceReturnsCorrectValueTest()[pri:0, instance:shop.CartTest@45c8e616]" name="verifyThatGetPriceReturnsCorrectValueTest" duration-ms="3" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyThatGetPriceReturnsCorrectValueTest -->
        <test-method status="PASS" signature="cleanCart()[pri:0, instance:shop.CartTest@45c8e616]" name="cleanCart" is-config="true" duration-ms="0" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanCart -->
      </class> <!-- shop.CartTest -->
      <class name="shop.RealItemTest">
        <test-method status="PASS" signature="verifyThatToStringReturnsCorrectResultTest()[pri:0, instance:shop.RealItemTest@4cdbe50f]" name="verifyThatToStringReturnsCorrectResultTest" duration-ms="1" started-at="2018-01-28T08:18:26Z" finished-at="2018-01-28T08:18:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyThatToStringReturnsCorrectResultTest -->
      </class> <!-- shop.RealItemTest -->
    </test> <!-- Regression1 -->
  </suite> <!-- Suite1 -->
</testng-results>
